infra/blueprint-test/pkg/tft/terraform.go

```diff
@@ -154,6 +164,11 @@ func NewTFBlueprintTest(t testing.TB, opts ...tftOption) *TFBlueprintTest {
 		}
 		tft.tfDir = tfdir
 	}
+
+	//set planFilePath
+	if tft.ShouldRunTerraformVet() {
+		tft.planFilePath = filepath.Join(tft.tfDir, "plan.tfplan")
```

Comment:
Any cons with writing this to a tmp dir that we clean up after `TerraformVet`

Reply:
fixed